cmd/eval-dev-quality/cmd/config.go

```diff
@@ -42,6 +59,16 @@ func (c *EvaluationConfiguration) Write(writer io.Writer) error {
 	return nil
 }
 
+// ReadEvaluationConfiguration reads an evaluation configuration file.
+func ReadEvaluationConfiguration(reader io.Reader) (configuration EvaluationConfiguration, err error) {
```

Comment:
Now that I am thinking of it... Can you please make `ConvertNamesToOSSpecific` unexported and add it after the reading so it happens automatically. And then move the other conversion (where we change all of them to use `/`) into the storing logic. That way someone using the configuration does not have to know that we internally always use `/`.